lib/ResolverFactory.js

```diff
@@ -412,9 +426,20 @@ exports.createResolver = function (options) {
 						"module"
 					)
 				);
+
 				plugins.push(
 					new PnpPlugin("raw-module", pnpApi, "undescribed-resolve-in-package")
 				);
+
+				// When PnpPlugin can't resolve a module, continue looking in node_modules
+				plugins.push(new NextPlugin("after-raw-module", "continue-raw-module"));
```

Comment:
This is the heart of the change I made that allows the `PnpPlugin` to call the `callback` without any arguments so that it can fallback to the node_modules resolution plugin (`ModulesInHierachicDirectoriesPlugin`).